### DISCUSSION TITLE
deepmdkit-lammps

### DISCUSSION THREAD

**Discussion by ShangZhe-1999** [deepmodeling/deepmd-kit#467] [General]
I installed DeepMDKit on the server of our school, and this server has been equipped with LAMMPS before. However, I saw on the official website that to connect LAMMPS with DeepMDKit, I need to download the LAMMPS installation package and install it again. Could you please tell me how to solve this problem?

**Comment by AnguseZhang** [comment]
Install via conda installation package in https://github.com/deepmodeling/deepmd-kit/releases , and an executable lammps compatible with DeePMD-kit will be installed.

**Comment by ShangZhe-1999** [comment]
Do you mean that I need to reinstall deepmd-kit    https://github.com/deepmodeling/deepmd-kit/releases/download/v1.3.3/deepmd-kit-1.3.3-cuda10.1_gpu-Linux-x86_64.sh ？
